packages/react/src/components/LoaderOverlay/index.tsx

```diff
@@ -7,37 +7,65 @@ import AxeLoader from './axe-loader';
 interface LoaderOverlayProps extends React.HTMLAttributes<HTMLDivElement> {
   variant?: 'large' | 'small';
   label?: string;
+  focus?: boolean;
+  children?: React.ReactNode;
 }
 
 const LoaderOverlay = React.forwardRef<HTMLDivElement, LoaderOverlayProps>(
-  ({ className, variant, label, ...other }: LoaderOverlayProps, ref) => (
-    <div
-      className={classNames(
-        'Loader__overlay',
-        className,
-        variant === 'large'
-          ? 'Loader__overlay--large'
-          : variant === 'small'
-          ? 'Loader__overlay--small'
-          : ''
-      )}
-      ref={ref}
-      {...other}
-    >
-      <div className="Loader__overlay__loader">
-        <Loader variant={variant} />
-        <AxeLoader />
+  (
+    {
+      className,
+      variant,
+      label,
+      children,
+      focus,
+      ...other
+    }: LoaderOverlayProps,
+    ref
+  ) => {
+    const overlayRef = createRef<HTMLDivElement>();
+
+    useEffect(() => {
+      if ((!!focus && overlayRef) || ref) {
+        setTimeout(() => {
```

Comment:
Trying to move focus prior to the next tick was causing issues where the styling for the focus indicator would not be applied, even thought he element was the active element.